src/instance.js

```diff
@@ -0,0 +1,5 @@
+import constructor from './constructor.js'
+
+export default function instance (typeOrNames) {
+  return new (constructor(typeOrNames))()
+}
```

Comment:
I can redo this if you'd like to recommend a style that better matches the rest of the codebase.  Can do the _.tap thing if you think it looks better.